ext/node/polyfills/01_require.js

```diff
@@ -605,10 +607,10 @@ Module._nodeModulePaths = function (fromPath) {
 Module._resolveLookupPaths = function (request, parent) {
   const paths = [];
 
-  if (ops.op_require_is_request_relative(request) && parent?.filename) {
+  if (ops.op_require_is_request_relative(request)) {
     ArrayPrototypePush(
       paths,
-      ops.op_require_path_dirname(parent.filename),
+      parent?.filename ? ops.op_require_path_dirname(parent.filename) : ".",
```

Comment:
Node returns `"."` when no filename is present. See: https://github.com/nodejs/node/blob/38e6ac7b446bcf2be257fde6763d1da7d2104295/lib/internal/modules/cjs/loader.js#L843-L849